examples/adb-lakehouse/README.md

```diff
@@ -0,0 +1,28 @@
+# Lakehouse terraform blueprints
+
+This example contains Terraform code used to provision a Lakehouse platform using the [adb-lakehouse module](../../modules/adb-lakehouse)
+
+## Deployed resources
+
+This example can be used to deploy the following:
+
+![Azure Lakehouse platform](https://raw.githubusercontent.com/databricks/terraform-databricks-examples/main/modules/adb-lakehouse/images/azure_lakehouse_platform_diagram.png?raw=true)
+
+* A new resource group
+* Networking resources including:
+  * Azure vnet
+  * The requiredt subnets for the Azure Databricks workspace.
+  * Azure route table (if needed)
+  * NSG
+* The Lakehouse platform resources, including:
+  * Azure Databricks workspace
+  * Azure key vault
+  * Azure data factory

```

Comment:
Should we encourage use of Data Factory?

Reply:
For Data ingestion from onprem and other sources. But module is customisable so it can be easily deleted. Thoughts ?

Reply:
`Azure data factory` is repeated two times. In the later designs we can make it customizable, like, create it only if you specify non-empty name, etc.